sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample10_SecretReference.md

```diff
@@ -0,0 +1,50 @@
+# Create, Retrieve and Delete a Secret Reference
+
+The AppConfiguration service supports KeyVault Secret reference settings.
```

Comment:
From an outsider, I guess it is not very clear what is the advantage of using this. Maybe make some emphasis on the way you can connect both services and retrieve/manage keys safely? or min interaction/modification?

Reply:
Added a few lines from docs.